### DISCUSSION TITLE
Go to Definition with Yarn PnP

### DISCUSSION THREAD

**Discussion by stewarthsoj** [zed-industries/zed#6559] [Feature Requests]
Hi,

Does Zed support Go to Definition with Yarn PnP at the moment? I've configured the `initialization_options` and according to the Zed logs it seems to be picking up the server, but I don't think Zed automatically knows how to handle zip archives. I don't get any errors in the file, although nothing happens on F12. Am I mistaken?

**Zed Logs**

```
2023-05-29T00:02:57 [INFO] unhandled notification window/logMessage:
{
  "type": 3,
  "message": "Using Typescript version (user-setting) 5.0.4-sdk from path \"/Users/****/Code/****/.yarn/sdks/typescript/lib/tsserver.js\""
}
```

**Zed Settings**

```
"lsp": {
    "typescript-language-server": {
        "initialization_options": {
            "tsserver": {
                "path": "./.yarn/sdks/typescript/lib"
            }
        }
    }
}
```

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
